index.html

```diff
@@ -716,8 +748,10 @@ <h1>Terminology</h1>
       <dt>
         <dfn>Virtual Thing</dfn>
       </dt>
-      <dd>An instance of a Thing that represents a Thing that is located
-        on another system component.</dd>
+      <dd>A <a>Service</a> that represents another <a>Thing</a>.
+       A Virtual Thing will often as as an <a>Intermediary</a>.
+       Examples include <a>Shadows</a> and <a>Digital Twins</a>.
```

Comment:
I too have previously used the term "Virtual Thing" to mean a Web Thing which has no physical device associated with it. E.g. WebThings Gateway has a "Virtual Things add-on" which lets you add all kinds of virtual devices to your gateway which don't exist in the real world. This is useful for testing, but also for use cases like creating a virtual on/off switch which controls multiple physical devices for example.

What would be a good term for describing a Web Thing which has no physical counterpart? Could this be another kind of Virtual Thing?

Reply:
hmm... you also use "virtual thing" for a "testing service" that "stands in place of" a physical thing.  I guess I could add that to the definition.  In this case, there is still "another thing" that the virtual thing is "standing in for".

Reply:
> In this case, there is still "another thing" that the virtual thing is "standing in for".

Not necessarily, it could be a virtual instance of a device which doesn't have a physical counterpart. As I said...

>> but also for use cases like creating a virtual on/off switch which controls multiple physical devices for example.

It still has all the properties of a Web Thing (a Thing Description, interaction affordances with network interfaces), it just doesn't represent an actual physical object in the real world. As far as a Consumer of a Web Thing is concerned the distinction is irrelevant, it doesn't know or care whether changes to the Web Thing manifest in the physical world.

Reply:
I meant by "standing in for" that the "virtual instance" was standing in for a "real instance".  Anyhow, please take a look at the updated definition.